app/views/handlers/individual_response.py

```diff
@@ -24,6 +24,10 @@
 GB_NIR_REGION_CODE = "GB-NIR"
 
 
+class IndividualResponseLimitExceeded(Exception):
+    pass
```

Comment:
I think this should be updated to `IndividualResponseLimitReached` be consistent with https://github.com/ONSdigital/eq-questionnaire-runner/blob/97dd209557e66c829728eae7c43b3e736ef8ad10/app/views/handlers/confirmation_email.py#L10-L11